src/main/java/org/sonarsource/sonarlint/ls/connected/ProjectBindingManager.java

```diff
@@ -166,38 +215,26 @@ private ProjectBindingWrapper computeProjectBinding(WorkspaceFolderSettings sett
   }
 
   @CheckForNull
-  public EndpointParamsAndHttpClient createServerConfiguration(String connectionId) {
+  public EndpointParamsAndHttpClient getServerConfigurationFor(String connectionId) {
     ServerConnectionSettings serverConnectionSettings = settingsManager.getCurrentSettings().getServerConnections().get(connectionId);
     if (serverConnectionSettings == null) {
       LOG.error("The specified connection id '{}' doesn't exist.", connectionId);
       return null;
     }
-    return getServerConfiguration(serverConnectionSettings);
+    return serverConnectionSettings.getServerConfiguration();
   }
 
-  public static class EndpointParamsAndHttpClient {
-    private EndpointParams endpointParams;
-    private ApacheHttpClient httpClient;
-
-    public EndpointParams getEndpointParams() {
-      return endpointParams;
-    }
-
-    public ApacheHttpClient getHttpClient() {
-      return httpClient;
-    }
-  }
-
-
   private Optional<ConnectedSonarLintEngine> getOrCreateConnectedEngine(
-    String connectionId, EndpointParamsAndHttpClient endpointParamsAndHttpClient, boolean autoUpdate, ProgressFacade progress) {
+    String connectionId, EndpointParamsAndHttpClient endpointParamsAndHttpClient, boolean autoUpdate, ProgressFacade
+    progress) {
     return connectedEngineCacheByConnectionId.computeIfAbsent(connectionId,
       s -> Optional.ofNullable(createConnectedEngineAndUpdateIfNeeded(connectionId, endpointParamsAndHttpClient, autoUpdate, progress)));
   }
 
   @CheckForNull
-  private ConnectedSonarLintEngine createConnectedEngineAndUpdateIfNeeded(String connectionId, EndpointParamsAndHttpClient endpointParamsAndHttpClient, boolean autoUpdate,
-    ProgressFacade progress) {
+  private ConnectedSonarLintEngine createConnectedEngineAndUpdateIfNeeded(String
```

Comment:
This is a rather weird formatting. Not blocker, but I suggest you have a look at the [SonarSource developer toolset documentation](https://github.com/SonarSource/sonar-developer-toolset#code-style-configuration-for-intellij).